<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>880</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(186,211,218);border:none;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,2,3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QComboBox" name="qSheetName">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#ffffff;</string>
          </property>
          <property name="currentText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">background-color:#ffffff;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="fileName">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">;background-color:#ffffff;</string>
          </property>
          <property name="text">
           <string>请选择文件</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QRadioButton" name="model1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#ffffff;</string>
          </property>
          <property name="text">
           <string>普  通</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QRadioButton" name="model2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#ffffff;</string>
          </property>
          <property name="text">
           <string>简  洁</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QPushButton" name="html">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">    color: white;
    background-color: #5d8aad;
    border: 1px #3079ED solid;
    box-shadow: inset 0 1px 0 #80B0FB;
    border-radius:5px;</string>
          </property>
          <property name="text">
           <string>html报告</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="debug">
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">    color: white;
    background-color: #5d8aad;
    border: 1px #3079ED solid;
    box-shadow: inset 0 1px 0 #80B0FB;
    border-radius:5px;</string>
          </property>
          <property name="text">
           <string>开  始</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="dtailLog">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">    color: white;
    background-color: #5d8aad;
    border: 1px #3079ED solid;
    box-shadow: inset 0 1px 0 #80B0FB;
    border-radius:5px;</string>
          </property>
          <property name="text">
           <string>查看日志</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="file">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">    color: white;
    background-color: #5d8aad;
    border: 1px #3079ED solid;
    box-shadow: inset 0 1px 0 #80B0FB;
    border-radius:5px;</string>
          </property>
          <property name="text">
           <string>选  择</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="analyJSON">
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">    color: white;
    background-color: #5d8aad;
    border: 1px #3079ED solid;
    box-shadow: inset 0 1px 0 #80B0FB;
    border-radius:5px;</string>
          </property>
          <property name="text">
           <string>解  析</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="dtailReport">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">    color: white;
    background-color: #5d8aad;
    border: 1px #3079ED solid;
    box-shadow: inset 0 1px 0 #80B0FB;
    border-radius:5px;</string>
          </property>
          <property name="text">
           <string>excel报告</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="1">
         <widget class="QLabel" name="failNum">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">height:23px;background-color:#ffffff;font: 12pt 'Arial';color: rgb(255,0,0);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_9">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">height:23px;background-color:#ffffff;</string>
          </property>
          <property name="text">
           <string>成  功</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_8">
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">height:23px;background-color:#ffffff;</string>
          </property>
          <property name="text">
           <string>异  常</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="result">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">height:23px;background-color:#ffffff;font: 12pt 'Arial';</string>
          </property>
          <property name="text">
           <string>0/0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">height:23px;background-color:#ffffff;</string>
          </property>
          <property name="text">
           <string>结果预览:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="skipNum">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">height:23px;background-color:#ffffff;font: 12pt 'Arial';color: rgb(248,172,89);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">height:23px;background-color:#ffffff;</string>
          </property>
          <property name="text">
           <string>失  败</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="successNum">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">height:23px;background-color:#ffffff;font: 12pt 'Arial';color: green;</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="refresh">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">    color: white;
    background-color: #5d8aad;
    border: 1px #3079ED solid;
    box-shadow: inset 0 1px 0 #80B0FB;
    border-radius:5px;</string>
          </property>
          <property name="text">
           <string>刷  新</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">background:white</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
